include/zephyr/dt-bindings/clock/stm32wl_clock.h

```diff
@@ -29,9 +29,10 @@
 /** Fixed clocks  */
 /* Low speed clocks defined in stm32_common_clocks.h */
 #define STM32_SRC_HSI		(STM32_SRC_LSI + 1)
+#define STM32_SRC_MSI		(STM32_SRC_HSI + 1)
 /* #define STM32_SRC_HSI48	TBD */
 /** Bus clock */
-#define STM32_SRC_PCLK		(STM32_SRC_HSI + 1)
+#define STM32_SRC_PCLK		(STM32_SRC_MSI + 1)
```

Comment:
Is changing this definition going to break anything for someone, considering now the value is 1 more than previous, if I am reading this correctly

Reply:
From what I understand this define is just used to attribute a name to a clock, so that it can be referenced in various places like the device tree. So as long as people have been using that define and not its value, that should be fine.

That said it would be nice to have the confirmation from someone knowing this part of the code

Reply:
Okay, I see, that's probably true, but yeah would like approval from a ST maintainer

Reply:
That's fine indeed